Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN ln -s /usr/local/bin/node /usr/local/bin/nodejs
 
 # install angular in a separate cached layer
 RUN mkdir -p /ddp-workspace/projects
-ENV ANGULAR_CLI @angular/cli@8.2.0
+ENV ANGULAR_CLI @angular/cli@8.3.21
 RUN npm install -g $ANGULAR_CLI
```

Comment:
@zyme We still using Dockerfile s for deployment or have we moved to using Google App Engine configurations?

Reply:
@zyme, can you help us? ^^

Reply:
Sorry for the delay!  Yes, we are still using Docker for our deploys--hopefully not too much longer, but we will need to keep doing these sorts of changes for now.